navigation/ca_hector/launch/geotiff_mapping.launch

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+
+<!-- Helper launchfile to store the map created with Hector SLAM in
+tif format.
+To save the map at any point of execution, you need to run:
+
+rostopic pub <ns>/syscommand std_msgs/String "savegeotiff"
+
+where ns is create + robot number (e.g. create1)
+-->
+
+<launch>
+  <!-- Path where the geotiff file will be stored -->
+  <arg name="geotiff_map_file_path" default="$(find ca_hector)/maps"/> -->
+
+  <include file="$(find ca_hector)/launch/default_mapping.launch"/>
+
+  <!-- Launch the geotiff mapper -->
+  <include file="$(find ca_hector)/launch/geotiff_mapper.launch">
+    <arg name="trajectory_source_frame_name" value="scanmatcher_frame"/>
```

Comment:
I think you have to put here the frame that corresponds to the laser. In our case `rplidar/laser_link` but I'm not completely sure so you should read the documentation (if it exists).

Reply:
@eborghi10 I couldn't find much in the documentation. Switching to that frame breaks Hector SLAM.